Automattic/Automattic-Tracks-iOS
# Update CocoaPods to 1.7.0 series

It's all in the title 😃

**To Test:**

Run `rm -rf Pods && bundle install && bundle exec pod install && bundle exec pod lib lint` on this branch to ensure that pods can be properly installed and built.
